### DISCUSSION TITLE
Navigation scrollbar and footer problem

### DISCUSSION THREAD

**Discussion by parag3** [squidfunk/mkdocs-material#2820] [Show and tell]
First of all really great theme developed. 

**Problem** - Have created a documentation site and have added lot of contents in to it. My Navigation has depth of 4 level and every level has multiple documents. But when i use setting for "navigation.expand" really get weird behavior for navigation and also for the footer.
* No scrollbars for navigation section
* Footer moves along with the scroll and comes in between.

I have replicated the issue by creating a sample new project and assigning the theme, added multiple navigation items under 1 section only. To replicate the problem, I did not even set the navigation settings, i did not even add any plugins also i did not even add any markdown extensions.

NOTE :: Not able to add issue hence added here.

** Steps to reproduce **
1. Create project using mkdocs 
2. open mkdocs.yml and assign theme
3. add multiple navigation items. More than 20.
4. run command "mkdocs serve" and site behaves properly
5. run command "mkdocs build" and go to site folder and open Index.html 
6. problems starts.

**Versions **
- Python: `3.9`
- MkDocs: `mkdocs, version 1.2.1`
- Material: `Version: 7.1.9`

**Configuration**
site_name: My Docs
site_url: https://example.com/

nav:   
    - "Home": 'index.md'
    - 'Menu 1': 
        - 'Menu 11': 'index.md'
        - 'Menu 11': 'index.md'
        - 'Menu 11': 'index.md'
        - 'Menu 11': 'index.md'
        - 'Menu 11': 'index.md'
        - 'Menu 11': 'index.md'
        - 'Menu 11': 'index.md'
        - 'Menu 11': 'index.md'
        - 'Menu 11': 'index.md'
        - 'Menu 11': 'index.md'
        - 'Menu 11': 'index.md'
        - 'Menu 11': 'index.md'
        - 'Menu 11': 'index.md'
        - 'Menu 11': 'index.md'
        - 'Menu 11': 'index.md'
        - 'Menu 11': 'index.md'
        - 'Menu 11': 'index.md'
        - 'Menu 11': 'index.md'
        - 'Menu 11': 'index.md'
        - 'Menu 11': 'index.md'
        - 'Menu 11': 'index.md'
        - 'Menu 11': 'index.md'
        - 'Menu 11': 'index.md'
        - 'Menu 11': 'index.md'
        - 'Menu 11': 'index.md'
        - 'Menu 11': 'index.md'
        - 'Menu 11': 'index.md'
        - 'Menu 11': 'index.md'
        - 'Menu 11': 'index.md'
        - 'Menu 11': 'index.md'
        - 'Menu 11': 'index.md'
        - 'Menu 11': 'index.md'
        - 'Menu 11': 'index.md'
        - 'Menu 11': 'index.md'
        - 'Menu 11': 'index.md'
        - 'Menu 11': 'index.md'
        - 'Menu 11': 'index.md'
        - 'Menu 11': 'index.md'
        - 'Menu 11': 'index.md'
        - 'Menu 11': 'index.md'
        - 'Menu 11': 'index.md'
        - 'Menu 11': 'index.md'

theme: 
    name: material
    font: false
    palette:
       primary: black
       secondary: red

- Operating system: windows 10
- Browser: chrome, Edge

![problem](https://user-images.githubusercontent.com/83712385/125195289-a3f7b080-e272-11eb-83da-3f3e6fb1516c.png)

**Comment by squidfunk** [comment]
This is likely related to the search breaking when you're browsing locally. If you open the console, there's likely some error talking about web workers. You can try Material for MkDocs 7.1.10 (latest version) which should gracefully degrade when browsing locally, as it will intercept search errors.

**Reply by parag3** [reply]
Thanks a lot for your response. Strange but worked.

I upgraded to 7.1.10 and both the issues are resolved (Navigation scrollbar and footer moving). As you suggested, yes there were 4 issues related to search json file. After the upgrade they still persist, but main issues are resolved for my local build.

I think once i deploy / host the build in git pages those errors might go. 

Will give it a try.
